bin/helpers/functions.sh

```diff
@@ -4,6 +4,7 @@
 function get_linker_ldflags {
     echo "
         -X 'github.com/mysterium/node/server.mysteriumApiUrl=${MYSTERIUM_API_URL}'
-        -X 'github.com/mysterium/node/communication/nats/discovery.natsServerIP=${NATS_SERVER_IP}'
+        -X 'github.com/mysterium/node/server.natsServerIP=${NATS_SERVER_IP}'
+        -X 'github.com/mysterium/node/client.mysteriumApiUrl=${MYSTERIUM_API_URL}'
```

Comment:
Why `mysteriumApiUrl ` several times

Reply:
its in different packages. to remove duplication I could make it public..